Source/JavaScriptCore/builtins/BuiltinNames.h

```diff
@@ -210,7 +211,7 @@ namespace JSC {
     macro(hasOwn) \
     macro(indexOf) \
     macro(pop) \
-    macro(asyncContext) \
+
```

Comment:
☝️ extra newline aligns with WebKit repo source (so closer to original source material).